app/controllers/admin/disputes/appeals_controller.rb

```diff
@@ -20,7 +20,7 @@ def reject
     authorize @appeal, :approve?
     log_action :reject, @appeal
     @appeal.reject!(current_account)
-    UserMailer.appeal_rejected(@appeal.account.user, @appeal)
+    UserMailer.with(user: @appeal.account.user).appeal_rejected(@appeal).deliver_later
```

Comment:
The spec for this action was only verifying that `:appeal_rejected` was called, not that delivery_later was called. This seems like a bugfix?